ios/FluentUI/Popup Menu/PopupMenuController.swift

```diff
@@ -31,6 +31,18 @@ open class PopupMenuController: DrawerController {
         static let descriptionVerticalMargin: CGFloat = 12
     }
 
+    public typealias TokenSetKeyType = PopupMenuTokenSet.Tokens
+    public typealias TokenSetType = PopupMenuTokenSet
+    public override var tokenSet: DrawerTokenSet {
+        get {
+            return popupTokenSet
+        }
+        set {
+            preconditionFailure("PopupMenuController tokens must be set through popupTokenSet")
```

Comment:
Do we need to stop execution in release builds here? I am leery of hard crashing unless we absolutely have to.

Reply:
Agreed -- this seems fine for a debug-only assert, but should probably resolve to a no-op in shipping code.